documentation/source/product-model.rst

```diff
@@ -683,6 +700,56 @@ The CDM supports Product Qualification functions for Credit Derivatives, Interes
 
 The output of the qualification function is used to populate the ``productQualifier`` attribute of the ``ProductTaxonomy`` object, which is created when a ``ContractualProduct`` object is created. The product taxonomy includes both the product qualification generated by the CDM and any additional product taxonomy information which may come from the originating document, such as FpML. In this case, taxonomy schemes may be associated to such product taxonomy information, which are also propagated in the ``ProductTaxonomy`` object.
 
+<<<<<<< HEAD
+Many different financial taxonomies may be used by various segments of the financial industry to describe the same product. To support a multitude of taxonomies without adding any specific identity to data types in the model, a Taxonomy type is used to identify the source and attributes any particular taxonomy structure.
+=======
+Many different financial taxonomies may be used by various segments of the financial industry to describe the same product. To support a multitude of taxonomies with adding and specific identity to data types in the model, a Taxonomy type is used to identify the source and attributes any particular taxonomy structure.
+>>>>>>> fe61dbde2 (icma-documentation-update)
+
+.. code-block:: Haskell
+ type Taxonomy:
+ 	source TaxonomySourceEnum (0..1)
+ 	value TaxonomyValue (0..1)
+	
+TaxonomyValue has been expanded to represent a complex type:
+
+.. code-block:: Haskell
+
+type TaxonomyValue: <"Defines a taxonomy value as either a simple string or a more granular expression with class names and values for each class.">
+	
+	name string (0..1) 
+		[metadata scheme]
+	classification TaxonomyClassification (0..*)
+	condition ValueExists:
+		name exists or classification exists
+	 
+TaxonomyClassification is also a complex type that support a hierarchical structure of any depth:
+
+ type TaxonomyClassification:
+	 className string (1..1)
+	 value string (1..1)
+	 description string (0..1)
+	 
+To represent the repurchase agreement using the CFI taxonomy the json would lookk like:
+
+.. code-block:: Javascript
+
+ "productTaxonomy": [
+ 	{
+		"source": "CFI",
+		"value": {
+			"name": {
+				"value": "LRSTXD"
+			}
+		}
+	},
+	{
+	"productQualifier": "Repurchase Agreement",
```

Comment:
to keep the same indentation these two lines should be indented by an additional tab